fetch_facebook_data.rb

```diff
@@ -11,6 +11,7 @@ def gets_data
     end
     return results
   end
+
   def filter(data_indexs)
```

Comment:
んー、このメソッド名はイケてないかもね。ソースコード読むまで何でフィルタするのか全然わからないから。
かつ、data_indexsとかいらないよね？ resultsをインスタンス変数に持たせちゃえば。
filter_by_date(since:, til:)的な感じが良いのでは。
そして、そうするならインスタンス変数の@from_time, @to_timeいらないね。